independent-publisher-connectors/Voice Monkey/README.MD

```diff
@@ -0,0 +1,76 @@
+# Voice Monkey
+
```

Comment:
Please update all required sections as per the template provided below.  
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md